.github/workflows/test-deploy.yml

```diff
@@ -90,6 +100,31 @@ jobs:
           --dry-run' }}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  binaries:
+    needs: authorize
+    name: binaries-${{ matrix.system }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.ref }}
+
+      - name: build
+        run: |
+          pkg -t node14-linux,node14-macos package.json
```

Comment:
```suggestion
        run: pkg -t node16-linux,node16-macos package.json
```

Reply:
Do we want `node16-win` not every command works. The `runner` for example and at least previous `publish` probably now it works because its wasm compiled